src/redux/auth/authReducers.ts

```diff
@@ -0,0 +1,12 @@
+import { PayloadAction } from "@reduxjs/toolkit";
+
+export const setTokenReducer = (
+  state: { token: string | null },
```

Comment:
`{ token: string | null }`  This code is repeated in this file, so it's better to put it in a separate interface above.